lib/dialects/postgres/query.js

```diff
@@ -309,6 +343,44 @@ module.exports = (function() {
         }
 
         break;
+      case '23P01':
+        // there are multiple different formats of error messages for this error code
+        // this regex should check at least two
+        match = errDetail.match(/Key \((.*?)\)=\((.*?)\)/);
+
+        if (match) {
+          fields = Utils._.zipObject(match[1].split(', '), match[2].split(', '));
+          errors = [];
+          message = 'Validation error';
+
+          Utils._.forOwn(fields, function(value, field) {
+            errors.push(new sequelizeErrors.ValidationErrorItem(
+              field + ' must be unique', 'unique violation', field, value));
+          });
+
+          if (this.callee && this.callee.__options && this.callee.__options.uniqueKeys) {
```

Comment:
Why are you using a unique error message and unique field checking for an Exclusion Constraint error? Am i missing something or is it leftover from copy-paste?